src/data.js

```diff
@@ -143,54 +143,56 @@ Namespace.ensureNamespace = function(namespace, expires) {
 /**Delete expired entries */
 Namespace.expireEntries = async function(now) {
   let continuationToken = undefined;
+  let count = 0;
+
   while (1) {
     let data = await this.scan({
       expires: Entity.op.lessThan(now),
     },
     {
-      limit:         500,
+      limit:         100,
       continuation:   continuationToken,
```

Comment:
hmm, unrelated, but you can do `handler(entry) {return entry.remove(); }` instead of this stuff too...

This would afaik do them all concurrently too... at-least `limit` at a time... so 100 at the same time...

Reply:
Yeah, this is a little bit more explicit about the parallelism.